### PR TITLE
Release: 2016.03.21.1

### LINKED ISSUE #77: users/<id>/stats endpoint change spotify_uri --> uri in track objects
This should be changed to match the rest of the api where the uri value in a track object is returned as `uri`

### PR DESCRIPTION
**Pull request by krak3n**
This should actually fix #77.

The main issue ended up being mixing 2 different types os serializer in the same resource, 1 for reading data from spotify and the other reading data from our database. The former uses `uri` and the later `spotify_uri` and thus have different sources for that data.

Simply swapped out the serialzers so the stats resource uses the DB serializer, returns the same JSON body but with more data.

### CODE REVIEW

**Comment by krak3n** [comment]
@thisissoon/members this should actually fix the problem

File: fm/views/user.py [hunk 1]
```diff
@@ -29,8 +29,9 @@
 from fm.thirdparty.spotify import (
     PlaylistSerializer,
     SpotifyApi,
-    TrackSerializer
+    TrackSerializer,
 )
+from fm.serializers.spotify import TrackSerializer as FMTrackSerializer
```
Comment (radeklos) [inline fm/views/user.py]: What is difference between `thirdparty.spotify.TrackSerializer` and `serializers.spotify.TrackSerializer`? Which is actually `FMTrackSerializer`?

Comment (krak3n) [inline fm/views/user.py]: One reads from the Spotify REST API, the other reads from our DB. Our the serializer for the DB instances is in `fm.serializers.spotify` and the Spotify REST API is in `fm.thirdpart.spotify` which I think you added to read data from spotify's API.

The views `fm.views.user` for returning user stats was doing DB I/O but not using the correct serializer for that task which is why it was returning data the FE didn't expect so I swapped them out and the view will now return the correct data.

The naming and module location here is a bit off and does require more thought in the long run and really this code base needs a refactor to bring it up to our API spec, but thats another PR for another day :wink:

**Comment by radeklos** [comment]
:+1:

**Pull request merged**